app/views/categories/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= stylesheet_link_tag "application" %>
+<%= stylesheet_link_tag "category" %>
+
+<%= render "layouts/nav_bar", {prev_path: nil, title: 'categories', from_form: false, next_path: categories_path, next_text: 'search' } %>
+
+<div class="categories">
+    <% if @categories.count > 0 %>
+        <% @categories.each do |category| %>
+            <%= render category %>
+        <% end %>
+    <% else %>
+        <small> No categories added, <%= link_to "click", new_category_path, class: 'clear-button' %> to add.</small>
+    <% end %>
+</div>
+
+<%= link_to raw("<i class='las la-plus'></i>"), new_category_path, class: 'primary-button floating-button button-round' if can? :create, Category %>
```

Comment:
- [ ] Please make sure to use the below styles for the `add new category` and `add new transaction` buttons to follow the given design guidelines.

![image](https://github.com/patriciachrysy/expenses-control/assets/83803414/47f0d1c5-8457-4470-88fc-0f0c32c029bf)